modules/robin-map/1.2.1/presubmit.yml

```diff
@@ -0,0 +1,13 @@
+matrix:
+  platform:
+  - centos7
+  - debian10
+  - ubuntu2004
+  - macos
+  - windows
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    build_targets:
+    - '@robin_map//:robin-map'
```

Comment:
this needs to be `@robin-map//:robin-map` (dash instead of underscore)

Reply:
Done.